pkg/controller/verifyapi/verify.go

```diff
@@ -41,6 +42,7 @@ import (
 func (c *Controller) HandleVerify() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := observability.WithBuildInfo(r.Context())
+		ctx = logging.WithLogger(ctx, c.logger.Named("handleverify"))
```

Comment:
Hmm - this should be:

```go
logger := c.logger.Named("HandleVerify")
```

and then update to use `logger` instead of `c.logger` in the rest of the method.

Reply:
removed - changed direction, don't need this